### ISSUE TITLE
Travis should give better feedback when a whitelisted package is not found because it requires a line in sources

### ISSUE THREAD

**Issue by JasonGross** [travis-ci/travis-ci#4597]
Quoting https://github.com/travis-ci/apt-package-whitelist/issues/620, it might be useful to have travis give instructions along the lines of "search here for the package, then add this or that line to soruces:" when the package is not found. Or, even better, print the particular source it's in, when it's not found.

**Comment by JasonGross** [comment]
I don't think this feature request has been granted (though I haven't checked carefully).

**Comment by JasonGross** [comment]
As far as I know, this still hasn't been fixed.

**Comment by BanzaiMan** [comment]
Finding a source that may provide the package is probably very difficult, but adding a blurb on the source should not be.
